pages/login.php

```diff
@@ -1,22 +1,19 @@
 <?php
 
-//some php code for verifying the user
-
-?>
+  //some php code for verifying the user
 
+;?>
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
```

Comment:
This one
Is this link included again at the bottom?

Reply:
no .

The link at the top is for css and at bottom its for js

Reply:
okay